src/plugins/correlations/server/saved_objects/correlations.ts

```diff
@@ -0,0 +1,41 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ *
+ * Any modifications Copyright OpenSearch Contributors. See
+ * GitHub history for details.
+ */
+
+import { SavedObjectsType } from '../../../../core/server';
+
+export const correlationsSavedObjectType: SavedObjectsType = {
```

Comment:
do we want to call this out as experimental and schema and APIs can change in near future